ci_scripts/docker-coredeps/deps/build_rust.sh

```diff
@@ -3,9 +3,9 @@
 set -e -x 
 
 # Install Rust 
-curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2019-04-19 -y
+curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2019-07-10 -y
```

Comment:
Would making this script read the toolchain from the `rust-toolchain` file in the root of the repo be a good idea?  No one is going to find this here to update when they update that file - I know I didn't.

Reply:
maybe -- right now we have no automated recreation of the docker image, and there is no caching of the compile -- the latter is the most pressing issue as the python tests take far too long to start running.